tests/test_config_parsing/test_enkf_obs_parsing.py

```diff
@@ -19,3 +26,540 @@ def test_that_enkf_obs_keys_are_ordered(tmp_path_factory, config_generator):
+                       {{
+                          START = {start};
+                          STOP  = {stop};
+                          ERROR = 0.50;
+                          ERROR_MODE = REL;
+                       }};
+                    }};
+                    """
+                )
+            )
+        with open("time_map.txt", "w", encoding="utf-8") as fo:
+            fo.writelines("2023-02-01")
+        run_sim(
+            datetime(2014, 9, 10),
+            [("FOPR", "SM3/DAY", None), ("FOPRH", "SM3/DAY", None)],
+        )
+
+        ert_config = ErtConfig.from_file("config.ert")
+
+        with pytest.warns(ConfigWarning, match=message):
+            _ = EnkfObs.from_ert_config(ert_config)
+
+
+@pytest.mark.parametrize(
+    "keys",
+    [
+        [("FOPR", "SM3/DAY", None), ("FOPRH", "SM3/DAY", None)],
+        [("WWIR", "SM3/DAY", "WNAME"), ("WWIRH", "SM3/DAY", "WNAME")],
+    ],
+)
+def test_that_history_observations_are_loaded(tmpdir, keys):
+    with tmpdir.as_cwd():
+        config = dedent(
+            """
+        NUM_REALIZATIONS 2
+
+        ECLBASE ECLIPSE_CASE
+        REFCASE ECLIPSE_CASE
+        OBS_CONFIG observations
+        """
+        )
+        with open("config.ert", "w", encoding="utf-8") as fh:
+            fh.writelines(config)
+        key, _, wname = keys[0]
+        local_name = key if wname is None else (key + ":" + wname)
+        with open("observations", "w", encoding="utf-8") as fo:
+            fo.writelines(
+                dedent(
+                    f"""
+                    HISTORY_OBSERVATION  {local_name}
+                    {{
+                       ERROR       = 0.20;
+                       ERROR_MODE  = RELMIN;
+                       ERROR_MIN   = 100;
+                    }};
+                    """
+                )
+            )
+        with open("time_map.txt", "w", encoding="utf-8") as fo:
+            fo.writelines("2023-02-01")
+        run_sim(datetime(2014, 9, 10), keys)
+
+        ert_config = ErtConfig.from_file("config.ert")
+
+        observations = EnkfObs.from_ert_config(ert_config)
+        assert [o.getKey() for o in observations] == [local_name]
+        assert observations[0].getNode(1).getValue() == 1.0
+        assert observations[0].getNode(1).getStandardDeviation() == 100.0
+
+
+def test_that_missing_time_map_raises_exception(tmpdir):
+    with tmpdir.as_cwd():
+        config = dedent(
+            """
+        JOBNAME my_name%d
+        NUM_REALIZATIONS 10
+        OBS_CONFIG observations
+        """
+        )
+        with open("config.ert", "w", encoding="utf-8") as fh:
+            fh.writelines(config)
+        with open("observations", "w", encoding="utf-8") as fo:
+            fo.writelines(
+                dedent(
+                    """
+                    GENERAL_OBSERVATION OBS {
+                       DATA       = RES;
+                       INDEX_LIST = 0,2,4,6,8;,
+                       RESTART    = 1;
```

Comment:
This is likely not a problem, but this value changed from 0 to 1 when moving this test.